src/hotspot/share/gc/z/zPageAge.hpp

```diff
@@ -45,6 +46,19 @@ enum class ZPageAge : uint8_t {
   old
 };
 
-constexpr uint ZPageAgeMax = static_cast<uint>(ZPageAge::old);
+constexpr uint ZPageAgeCount = static_cast<uint>(ZPageAge::old) + 1;
+constexpr ZPageAge ZPageAgeLastPlusOne = static_cast<ZPageAge>(ZPageAgeCount);
+
+ENUMERATOR_VALUE_RANGE(ZPageAge,
+                       static_cast<uint>(ZPageAge::eden),
+                       ZPageAgeCount);
+
```

Comment:
Could this be using the other define to set this up?
```suggestion
ENUMERATOR_VALUE_RANGE(ZPageAge,
                       ZPageAge::eden,
                       ZPageAge::old);
```
Or does that not work?

Reply:
Yes, using the other define (`ENUMERATOR_RANGE`) with the enum type instead of the underlying type makes more sense.